src/main/java/org/folio/rest/impl/PatronServicesResourceImpl.java

```diff
@@ -465,16 +472,18 @@ private Account updateItem(Charge charge, Item item, Account account) {
     return account;
   }
 
-  private int getLimit(boolean includeItem) {
-    final int limit;
-
-    if (includeItem) {
-      limit = Integer.MAX_VALUE;
-    } else {
-      limit = 1; // until RMB-96 is implemented, then 0
+  private Map<String, String> getLimitAndOffsetParams(int limit, int offset, boolean includeItem) {
+    if (!includeItem) {
+      return Map.of(
+        "limit", "0");
     }
-
-    return limit;
+    if (limit > 0) {
+      return Map.of(
+        "limit", String.valueOf(limit),
+        "offset", String.valueOf(offset));
+    }
+    return Map.of(
```

Comment:
FOLIO is inconsistent w/ how limits < 0 are handled.  In some cases a 400 is returned with a text/plain error message.  In other cases the limit param is ignored.

In this case what you have here is consistent with not specifying a limit at all, which I think makes sense.